include/boost/beast/_experimental/test/impl/stream.hpp

```diff
@@ -77,6 +77,16 @@ class basic_stream<Executor>::read_op : public detail::stream_read_op_base
           return net::get_associated_allocator(h_);
         }
 
+        using cancellation_slot_type =
```

Comment:
do we need to support cancelation for the test stream?

Reply:
I am using tcp streams on purpose so we don't.

Reply:
but it looks like we are adding cancelation to the test stream here?